include/dgl/runtime/tensordispatch.h

```diff
@@ -180,6 +215,10 @@ class TensorDispatcher {
     static constexpr int kCUDARawDelete = 3;
     static constexpr int kCUDACurrentStream = 4;
     static constexpr int kRecordStream = 5;
+    static constexpr int kCUDARawHostAlloc = 6;
+    static constexpr int kCUDARawHostDelete = 7;
+    static constexpr int kCUDARecordHostAlloc = 8;
+    static constexpr int kCUDAHostAllocEmptyCache = 8;
```

Comment:
Two 8's?

Reply:
Sorry for my typo. This is scary at first glance... I guess `reinterpret_cast` 'saved' me to make it work even they shared the same entry point.
Thanks, I fixed it.